notebooks/WatsonNLP_Demo.ipynb

```diff
@@ -0,0 +1,514 @@
+{
```

Comment:
<p>Instead of printing out the text representation, the notebook should display each of these dataframes using Jupyter's support for viewing dataframes as HTML tables. One cell/output per dataframe.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>